### DISCUSSION TITLE
Installation and Configuration

### DISCUSSION THREAD

**Discussion by philippkratzer** [kodebach/hacs-idm-heatpump#80] [Q&A] [answered]
Hey!

I am struggeling with the installation of the integration. This is the first time I am using a HACS integration.

So far I installed HACS, added the custom repository and downloaded the "Idm Heatpump Integration". It also shows up in the HACS integrations section.

I am wondering what the next steps are. According to the documentation "Configuration is done in the UI", which i am not able to find anywhere, without additional information.

Can anybody help please? Thank you very much!

**Comment by KVS62** [comment, marked as answer]
This post gives some more instructions, so hope that this will get you further:

https://github.com/kodebach/hacs-idm-heatpump/issues/26

**Reply by philippkratzer** [reply]
Ah great thanks! That helped a lot.

I was not aware that i need to add the integration after installing it in HACS.
The missing part was

> Open your Home Assistant page manually and then go to "Settings" -> "Devices & Services".
> Click the "Add Integration" button in the bottom right corner.
> Search for and select "IDM heat pump" and complete the setup steps.

This makes sense now, maybe it was already too late yesterday :)
Thanks a lot

**Reply by kodebach** [reply]
Ah yes, for new users it is not clear that once installed via HACS, the integration acts like any other Home Assistant integration. I'll at that to the README in the next release.
